tests/test_config.py

```diff
@@ -65,12 +65,27 @@ def test_proxy_headers():
     assert isinstance(config.loaded_app, ProxyHeadersMiddleware)
 
 
-def test_app_unimportable():
+def test_app_unimportable_module():
     config = Config(app="no.such:app")
     with pytest.raises(ImportError):
         config.load()
 
 
+def test_app_unimportable_other(caplog):
+    config = Config(app="test_config:app")
+    with pytest.raises(SystemExit):
+        config.load()
+        error_messages = [
```

Comment:
These lines need to be dedented, right now they most likely don't run because of the SystemExit exception thrown by config.load

Reply:
you're absolutely correct, my bad !